pom.xml

```diff
@@ -150,6 +154,34 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <artifactId>maven-jar-plugin</artifactId>
+        <executions>
+          <execution>
+            <!-- Produce a jar with a platfrom-specific classifier. -->
```

Comment:
More accurately: "Make the default target the platform-specific jar: only the shade plugin should produce a package without a classifier"